docs/concepts/bounded-context.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Bounded Context Concept
```

Comment:
No need to have the ”Concept” suffix. Please remove here and in other concept pages.